test/e2e/network/netpol/kubemanager.go

```diff
@@ -181,12 +185,22 @@ func (k *kubeManager) probeConnectivity(args *probeConnectivityArgs) (bool, stri
 	}
 
 	commandDebugString := fmt.Sprintf("kubectl exec %s -c %s -n %s -- %s", args.podFrom, args.containerFrom, args.nsFrom, strings.Join(cmd, " "))
-	stdout, stderr, err := k.executeRemoteCommand(args.nsFrom, args.podFrom, args.containerFrom, cmd)
-	if err != nil {
-		framework.Logf("%s/%s -> %s: error when running command: err - %v /// stdout - %s /// stderr - %s", args.nsFrom, args.podFrom, args.addrTo, err, stdout, stderr)
-		return false, commandDebugString, nil
+	probeRetryDelay := args.retryDelay
+	for i := 0; i < args.maxAttempts; i++ {
+		stdout, stderr, err := k.executeRemoteCommand(args.nsFrom, args.podFrom, args.containerFrom, cmd)
+		if err != nil {
+			framework.Logf("attempt %d: %s/%s -> %s: error when running command: err - %v /// stdout - %s /// stderr - %s", i, args.nsFrom, args.podFrom, args.addrTo, err, stdout, stderr)
+
+			// wait before next attempt
+			time.Sleep(time.Duration(probeRetryDelay) * time.Second)
```

Comment:
normally would do exponential backoff here but isnt this linear ? that i think would reduce CPU usage, and likely give us a more forgiving probe recovery in worst case scenario ?